pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+Final PHEPs can also be replaced by a different PHEP, rendering the original obsolete.
+
+The possible paths of the status of PHEPs are as follows:
+
+![PHEP status flow diagram](phep-0001/process_flow.svg)
+
+While not shown in the diagram, "Accepted" PHEPs may technically move to "Rejected" or "Withdrawn" even after acceptance. This will only occur if the implementation process reveals fundamental flaws in the design that were not noticed prior to acceptance of the PHEP.
+
+Upon reaching "Final", "Withdrawn", or "Rejected" status, the editors update the preamble and merge the PR into `main`.
+
+## PHEP Maintenance
+<a name="phep-maintenance"></a>
+PHEPs are no longer substantially modified after they have reached the Accepted, Final, Rejected, Withdrawn, or Replaced state. Once resolution is reached, a PHEP is considered a historical document rather than a living specification and the PHEP number is a permanent identifier for the document.
+
+If changes based on implementation experience and user feedback are made to PHEPs while in the Accepted state, they should be noted in the PHEP, such that the PHEP accurately describes the implementation at the point where it is marked Final.
+
+Rejected and Withdrawn PHEPs must not be resurrected but may be used as the basis for a new PHEP.
+
+Changes to PHEPs in the Final state are limited to the preamble, and only in the event the PHEP is replaced. Final PHEPs are immutable and must be replaced by new PHEPs if necessary, rather than updated. The scope of a PHEP should be carefully considered in this light.
+
+Preamble changes are made via PR to the `standards` repository, merged by PHEP editors.
+
+# What belongs in a successful PHEP?
+Each PHEP should have the following parts/sections:
+
+1. Preamble - [RFC 2822](https://datatracker.ietf.org/doc/html/rfc2822.html) style headers containing meta-data about the PHEP, including the PHEP number, a short descriptive title (limited to a maximum of 44 characters), the names, and optionally the contact info for each author, etc.
+2. Abstract - a short (~200 word) description of the technical issue being addressed.
+3. Motivation - The motivation is critical for PHEPs that want to change the PyHC ecosystem. It should clearly explain why the existing standards are inadequate to address the problem that the PHEP solves. This can include collecting documented support for the PHEP from important projects in the PyHC ecosystem. PHEP submissions without sufficient motivation may be rejected.
+4. Rationale - The rationale fleshes out the specification by describing why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages.
```

Comment:
Would it also make this section more readable if we bolded the 13 keywords (**Preamble**, **Abstract**, **Motivation**, etc)?

Reply:
I'm going to stick with ~ although I like the "well it's sort of about BUT NO MORE THAN" of ≲.

Readability: I think this is probably the worst part of the whole shebang. Let's take up that discussion in [Nick's comment](https://github.com/heliophysicsPy/standards/pull/22#pullrequestreview-1695499770) since I think there's bigger pruning to be done first and then we can think of formatting improvements?

Reply:
Valid. "~200" is easier to read.

Reply:
I don't think my edits improved the readability enough on their own, so I did bold those keywords (will be in next push).

Reply:
Bolded. I think that was everything in this thread, so resolving.